package.json

```diff
@@ -31,17 +31,19 @@
     "@bcrs-shared-components/court-order-poa": "3.0.11",
     "@bcrs-shared-components/date-picker": "1.3.3",
     "@bcrs-shared-components/document-delivery": "1.2.0",
+    "@bcrs-shared-components/document-id": "^1.0.5",
     "@bcrs-shared-components/effective-date-time": "1.1.15",
-    "@bcrs-shared-components/enums": "1.1.11",
+    "@bcrs-shared-components/enums": "^1.1.23",
```

Comment:
Please remove `^` and re-run npm install.

As a general rule, we want to use fixed/known versions of the shared components, otherwise an update might change something and we wouldn't know about it.

Reply:
Fixed, thanks.